umh-core/Makefile

```
@@ -511,3 +511,17 @@ build-push: build push-platform
 # Retrieve metrics from the UMH Core instance and save them to a file
 dump-metrics:
 	curl -s http://localhost:8081/metrics > metrics.txt
+
+run: 
```

checkmake (0.2.2)

[warning] Target body for "run" exceeds allowed length of 5 (12).

(maxbodylength)